src/10-sign-container.md

```diff
@@ -47,7 +47,7 @@ export SIGSTORE_ROOT_FILE="$HOME/fulcio-root.pem"
 Download the Rekor public key:
 
 ```bash
-wget -O publicKey.pem https://rekor.ozz-sigstore-instance.dev/api/v1/log/publicKey
+wget -O publicKey.pem https://rekor.sigstore.dev/api/v1/log/publicKey
```

Comment:
Hey! Thanks a lot for fixing things. The intent is for folks to download the public key from the rekor instance they're deploying, so "https://rekor.example.com" would be more appropriate. Thanks for pointing out this typo!

Reply:
Right, thanks! Changed in https://github.com/lukehinds/sigstore-the-hard-way/pull/32/commits/0c20f6e6991d859d59cef2754008271d62324e71